posts/2022/sparse-PFT-gridding.ipynb

```diff
@@ -0,0 +1,12504 @@
+{
```

Comment:
<p>Might want to add a bit more here - an overview? Or repurpose some of the content you mentioned at the beginning</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
:+1: I'm  hoping to convince someone from the land group to contribute a few lines. I've sent out anemail...

Reply:
<p>:+1:</p>